bessctl/commands.py

```diff
@@ -992,6 +1000,21 @@ def command_module(cli, module, cmd, arg_type, args):
         cli.bess.resume_all()
 
 
+@cmd('command gatehook GATEHOOK MODULE DIRECTION GATE GATEHOOK_CMD ARG_TYPE [CMD_ARGS...]',
+     'Send a command to a gatehook')
+def command_gatehook(cli, name, module, direction, gate, cmd, arg_type, args):
+    if args is None:
+        args = {}
+
+    cli.bess.pause_all()
```

Comment:
Should we always pause here? I guess we do for all commands, it's only the add module/connection code that has a conditional pause. It's definitely not _wrong_ anyway.

Reply:
Right - I think it would be good to have, in the interface that collects commands, a flag for "this command can be run while workers run" and then bessctl could use that to control its auto-pause, for instance. But that's way beyond this PR...